portality/models/v2/journal.py

```diff
@@ -980,9 +997,12 @@ class IssnQuery(object):
         }
     }
 
-    def __init__(self, owner):
+    def __init__(self, owner, in_doaj=False):
```

Comment:
Does adding this default to `False` change any other usages of this query?  The previous version didn't consider `in_doaj` at all, so returned everything.  Now we will definitely only return in_doaj=True or in_doaj=False values.

Also, if the default value is valid at all, the default to `False` seems risky.  You should probably have to intentionally ask for things that are not in DOAJ, otherwise you should just get the stuff that is.